packages/tradershub/src/features/cfd/modals/RegulationModal/Row.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { qtMerge } from '@deriv/quill-design';
+import { Text, TextProps } from '@deriv-com/ui/dist/components/Text';
+import { TRegulatorsContentProps, TRowItem } from '../../../../constants/regulators-modal-content';
+
+const Row = ({ attribute, content, id, idx }: TRegulatorsContentProps) => (
+    <tr className={qtMerge('min-h-2000', idx === 0 && 'bg-system-light-red-light-background')} key={id}>
+        <td
+            className={`sticky z-10 align-middle border-solid start-50 py-500 px-400 border-system-light-active-background border-x-75 border-b-75 ${
+                idx === 0 ? 'bg-system-light-red-light-background' : 'bg-system-light-primary-background'
+            }
+            `}
+        >
+            <Text align='left' as='p' color='general' size='xs' weight='bold'>
+                {attribute}
+            </Text>
+        </td>
+        {Object.keys(content).map(rowKey => (
+            <td
+                className='flex-col justify-center align-middle border-solid whitespace-nowrap py-500 px-400 border-b-75 border-r-75 border-system-light-active-background'
+                key={rowKey}
+            >
+                {Array.isArray(content[rowKey]) ? (
+                    (content[rowKey] as TRowItem[])?.map(item => (
```

Comment:
Convert that function into a constant and you dont need this type anymore

Reply:
@aizad-deriv when I convert the utility function to an array it didn't change anything with these types, I still need use `as` for `TRowItem[]` and for `TRowItem`. I tried to use `content[rowKey] instanceof Array` instead of `Array.isArray` but the result is the same. I'll try to find another solution but for now just like this

Reply:
Also tried `'text' in content[rowKey]` for `TRowItem`, but it doesn't work too...

Reply:
Alright then maybe stick to having it with object keys because in my opinion having a function then mapping is a bit weird. Don't you think?

Reply:
@aizad-deriv Honestly, I didn't understand what you mean. Cause of this issue is `Array.isArray` doesn't narrow the types (from `RowItem[] | RowItem` to just `RowItem[]`) and in result I have to show the TS that it's really array, and I need to use `as` for this